### PR TITLE
[WNMGDS-1904][WNMGDS-1905][WNMGDS-1906] Add one column page and story

### PR DESCRIPTION
**Pull request by line47**
## Summary

- Added a one column page layout for Healthcare.gov to the doc site
- Added a one column page layout story to storybook
- Embedded storybook story in doc site page
- Added a top level layouts navigation item for page layouts to live under
- https://jira.cms.gov/browse/WNMGDS-1904
- https://jira.cms.gov/browse/WNMGDS-1905
- https://jira.cms.gov/browse/WNMGDS-1906

## How to test

1. Build the healthcare storybook with `yarn build-storybook:gatsby` 
2. Run the hcgov storybook `yarn storybook:healthcare` and visit the one col page layout under patterns.
3. Run the doc site with `yarn start` and visit the one col page layout under layouts 

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title`

### If this is a change to code:

- [x] Created or updated unit tests to cover the logic added
- [x] Updated unit test snapshots (`yarn update-snapshots`) and loki reference images **if necessary** (`yarn loki && yarn loki:healthcare && yarn loki:medicare`)

### If this is a change to documentation:

- [x] Checked for spelling and grammatical errors

### Screenshots
Screen Shot 2022-09-20 at 12 45 23 PM

Screen Shot 2022-09-20 at 11 42 52 AM

### CODE REVIEW

**Comment by brentschneider** [comment]
@line47 content is ready for your review.  
@pwolfert Think I'm ready for that CI test magic you mentioned.

**Comment by forrestbaer** [comment]
Could you update PR with updates on what changed and steps for testing? Thank you.

File: .storybook/storybookStyles.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 // any @font-face declarations need to be included everywhere to not be
 // wrapped with the data-theme selector below
 @import '@cmsgov/ds-medicare-gov/src/styles/_fonts';
+@import '../packages/docs/src/styles/components/embeddedExamples.scss';
```
[the remark this answers is not in the document]
Comment (line47) [reply .storybook/storybookStyles.scss:5]: This was added to pull in the styles for the grid if I'm not mistaken CC @pwolfert

**Review by forrestbaer — APPROVED** [review]
Looks good to me! 👍🏻

**Comment by line47** [comment]
Hey @brentschneider made a few minor content adjustments, whenever you can take a look would be :100:

**Review by brentschneider — APPROVED** [review]
Excellent content additions, Scott!

**Comment by line47** [comment]
@pwolfert Help! This PR keeps failing when i restart in jenkins

**Pull request merged**